POS/Integration/03_ConfigurePrinter.md

```diff
@@ -21,9 +22,9 @@ The current APS is installed and has been started including a successful login.
 ![Assign printers](../../Assets/Screenshots/Printing/Settings/AssignPrinters.png "[Assign printers]")
 
 1. Click the *Global (for all groups)* printer in the list of printers.   
-    The *Settings for group Global* view is displayed.
+    The *Print Settings for group Global* view is displayed.
```

Comment:
*Print Settings for group Global* -> ?? Im Screenshot steht *Settings for group Global*

Reply:
done